uint16 id
uint8  dlc     # データ長
uint8[8] data